model/Signal.py

```diff
@@ -26,7 +26,7 @@ def __init__(self, path_to_file=None, signal=None, step=10, window_size = 50):
         
     def get_features(self, list_features = "all", remove_transition=False):
         if remove_transition:
-            self.remove_transition()
+            self.remove_transition(w=40) 
```

Comment:
don't change things in other functions. Isolate the functionality you're adding.